### ISSUE TITLE
Can't dump IPA - Unable to connect to port 2222 on 127.0.0.1 or ::1

### ISSUE THREAD

**Issue by VanHoevenTR** [noobpk/frida-ios-hook#67]
**Describe the bug**
It always trying to connect to wrong IP and port if I try to dump regardless if I specified the IP and port in hook.js

```
[24-11-22 16:34:45] [ERROR] - [x_x] Something went wrong, please check your error message.
 Message - can only concatenate str (not "int") to str
[24-11-22 16:34:45] [INFO] - [*] Dumping...
[Errno None] Unable to connect to port 2222 on 127.0.0.1 or ::1
```

But app listing and SSH works fine, so the IP and port are correct

**To Reproduce**
1. `python3 setup.py`
2. `python3 ioshook.py -d -p (package)`

**Desktop (please complete the following information):**
 - OS: Windows
 - Version 11

**Smartphone (please complete the following information):**
 - Device: iPad 9
 - OS: iOS 14.6

**Comment by noobpk** [comment]
This option required `iproxy` and Windows is not supported.
On Linux/macOS: You need to connect the device to the laptop and setup `iproxy 2222 22` . 
Reference: https://iphonedev.wiki/index.php/SSH_Over_USB

**Comment by noobpk** [comment]
try using attach mode instead of spawn mode

**Comment by noobpk** [comment]
my dump option base on https://github.com/AloneMonkey/frida-ios-dump . you can try using https://github.com/AloneMonkey/frida-ios-dump

**Comment by noobpk** [comment]
Yes. Nothing is perfect. Maybe the app has mechanisms to detect frida and block it.

**Comment by Kediel** [comment]
> I'm using it but it crashes on some apps. But nevermind, my friend tried it and it worked for him with specific apps

What specific apps, none of them are working on my iPhone 6.

**Comment by Kediel** [comment]
> Neural Cloud, and others I dont remember. I think it has to do with substrate and jb. It didn't work on my iPad 9 with checkra1n and substrate while it's working on my friends iPhone with unc0ver and substitude

I got somewhere by using https://github.com/KJCracks/Clutch on my older Macbook (x64_x86) to get the .ipa for Neural Cloud and Facebook. However, it was an incomplete dump so I had to export the .ipa quickly before it was erased from the folder. Leaving me with frameworks only. I would love to help bring this up to speed but would need time and communication with those more involved than I am.
